styles/pythonanywhere/catppuccin.user.css

```diff
@@ -0,0 +1,1538 @@
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0;
+      }
+    }
+
+    /* Ace file editor */
+    .ace_editor {
+      background-color: @crust;
+      color: @text;
+
+      .ace_gutter {
+        background: @mantle;
+        color: @overlay1;
+      }
+      .ace_print-margin {
+        background: @mantle;
+      }
+
+      .ace_marker-layer {
+        .ace_active-line {
+          background-color: fade(@yellow, 20%);
+        }
+        .highlight-line-error {
+          background-color: fade(@red, 20%);
+        }
+        .ace_bracket {
+          border-color: @overlay1;
+        }
+      }
+
+      .ace_cursor {
+        color: @text;
+      }
+      .ace_marker-layer .ace_selection {
+        background: fade(@accent-color, 30%);
+      }
+      .ace_fold {
+        background-color: @surface0;
+        border-color: @surface1;
+      }
+
+      .ace_constant.ace_language,
+      .ace_keyword,
+      .ace_meta,
+      .ace_variable.ace_language {
+        color: @mauve;
+      }
+      .ace_line .ace_identifier:not(:first-of-type) {
+        color: @blue;
+      }
+      .ace_paren {
+        color: @red;
+      }
+      .ace_constant.ace_numeric {
+        color: @peach;
+      }
+      .ace_entity.ace_other.ace_attribute-name,
+      .ace_support.ace_constant,
+      .ace_support.ace_function {
+        color: @teal;
+      }
+      .ace_entity.ace_name.ace_tag,
+      .ace_support.ace_class,
+      .ace_support.ace_type {
+        color: @blue;
+      }
+      .ace_storage {
+        color: @peach;
+      }
+      .ace_string {
+        color: @green;
+      }
+      .ace_comment {
+        color: @overlay2;
+      }
+
+      .ace_indent-guide {
+        @svg: escape(
+          '<svg width="1" height="2" xmlns="http://www.w3.org/2000/svg"><path stroke="@{surface0}" d="M0 0v2"/></svg>'
+        );
+
+        background-image: url("data:image/svg+xml;utf8,@{svg}");
+      }
+    }
+
+    .ace_tooltip {
+      background-color: @crust;
+      color: @text;
+      border-color: @surface0;
+    }
+
+    #ace_settingsmenu {
+      background-color: @mantle;
+      box-shadow: none;
+      color: @subtext0;
+    }
+
+    #kbshortcutmenu {
+      background-color: @mantle;
+      box-shadow: none;
+
+      .ace_optionsMenuEntry {
+        &:hover,
+        &:focus {
+          background-color: @base;
+        }
+      }
+
+      &,
+      .ace_optionsMenuCommand {
+        color: @subtext0;
+      }
+
+      .ace_optionsMenuKey {
+        color: @accent-color;
+      }
+    }
+
+    /* Console */
+
+    x-screen {
```

Comment:
The console style only seems to work on chrome:

![image](https://github.com/user-attachments/assets/2816612d-48ec-4d29-95b6-96e4f92514cf)

Reply:
Guessing that is either an issue with your Stylus settings or with how Stylus applies to iframes on Firefox. Not sure it is within our control.

Reply:
my guess is that it has something to do with the iframe source url being `#`

Reply:
Feel free to raise an issue on https://github.com/openstyles/stylus about it, I don't see any open issues about it.

Reply:
update: no clue it's so strange